plugins/module_utils/pure.py

```diff
@@ -21,13 +21,15 @@
 
 HAS_PURITY_FB = True
 try:
-    from purity_fb import PurityFb, FileSystem, FileSystemSnapshot, SnapshotSuffix, rest
+    from purity_fb import PurityFb, FileSystem, FileSystemSnapshot, SnapshotSuffix, rest  # noqa: F401, pylint: disable=unused-import
```

Comment:
FileSystem, FileSystemSnapshot and SnapshotSuffix really look like unused imports. I couldn't find it anywhere else in plugins/.

Reply:
Same here.